main.cpp

```diff
@@ -358,23 +360,23 @@ struct PowerSupply
     void setPowerState(bool powerState);
     void setStandbyState(bool standbyState);
     void blowFuse(); //{ fuseState = false;}
-private:
-    bool _powerState = 1;
-    bool _standbyState = 1;
+
+    bool powerState = 1;
+    bool standbyState = 1;
```

Comment:
Always initialize boolean variables with `true/false`.
never use integers to initialize them.
- [ ] revise